prometheus-exporters/openstack-exporter/alerts/cinder.alerts

```diff
@@ -78,7 +78,7 @@ groups:
       severity: info
       tier: vmware
       service: cinder
-      support_group: compute-storage-api
+      support_group: compute
       context: "openstack-exporter"
       meta: "Cinder pool {{ $labels.shard }}/{{ $labels.backend }}/{{ $labels.pool }} is overcommited"
       playbook: docs/support/playbook/cinder/cinder-pool-down.html
```

Comment:
playbook points to pool-down for an "overcommited" alert

Reply:
David told me it had to be 'compute' in slack yesterday, so I'm quite confused.

Reply:
it's probably correct. I just cannot verify it myself, because the docs have contradicting information. I would assume the docs are wrong here, tbh, but need Tilo to verify this.

Without being able to verify this independently, I cannot give approval. Feel free to merge anyways :)

Reply:
> playbook points to pool-down for an "overcommited" alert

yes, because the pool is down.   And on the pool down documentation there are 2 possible solutions documented that describes what to do when it's down because it's overcommited.

Reply:
ah. right. things go down when overcommited. forgot that. thanks.